src/app/loadout-builder/filter/EnergyOptions.tsx

```diff
@@ -81,8 +83,16 @@ export default function EnergyOptions({
         },
       },
       {
-        label: t('LoadoutBuilder.Masterworked'),
-        tooltip: t('LoadoutBuilder.LockElementOptions.Masterworked'),
+        label:
+          optimizingLoadoutName !== undefined
+            ? t('LoadoutBuilder.InOtherLoadouts')
+            : t('LoadoutBuilder.InLoadouts'),
+        tooltip:
+          optimizingLoadoutName !== undefined
+            ? t('LoadoutBuilder.LockElementOptions.InOtherLoadouts', {
+                loadoutName: optimizingLoadoutName,
+              })
+            : t('LoadoutBuilder.LockElementOptions.InLoadouts'),
```

Comment:
Do we need both of these? I feel like we could just have a single translation "In a loadout" and I don't really see the value in having both. If you have a loadout being optimised a given item may or may not be from that loadout and be having its energy locked.

Reply:
I'm afraid I don't understand what you're asking. Is this purely about the tooltips and descriptions and are you suggesting the "In Loadouts" description is good enough or are you suggesting armor equipped in the loadout I'm currently optimizing should have its element locked with this option?

Because the intended workflow is that if a piece of armor is equipped in exactly one loadout and I click "Optimize Armor" on that loadout, this option should not lock the element because changing the element on the piece and then overwriting the loadout can't break other loadouts, and the localization was meant to make that aspect really clear.

Reply:
Sorry I should have been clearer. I don't think we need the differing translations for each depending on whether the `optimizingLoadoutName` is undefined or not.

Reply:
I also think I missed something regarding that too so maybe my comment is void. Give me a few mins to read over it again.

Reply:
Yup ok, I get it all now. Can disregard this comment.